app/build.gradle

```diff
@@ -32,4 +32,7 @@ dependencies {
     compile 'com.android.support:appcompat-v7:25.3.1'
     testCompile 'junit:junit:4.12'
     compile 'com.android.support.constraint:constraint-layout:1.0.2'
+    compile "com.github.kittinunf.fuel:fuel:$fuel_version" //for JVM
```

Comment:
Following the documentation, `fuel` and `fuel-rxjava` are maybe not necessary in our case